src/aotcompile.cpp

```diff
@@ -1854,7 +1854,7 @@ void jl_get_llvmf_defn_impl(jl_llvmf_dump_t* dump, jl_method_instance_t *mi, siz
         // Force at least medium debug info for introspection
         // No debug info = no variable names,
         // max debug info = llvm.dbg.declare/value intrinsics which clutter IR output
```

Comment:
This comment is wrong (since #50585), since we are now emitting the max debug info

Reply:
So this is where this is a mess. Because we have the two settings, and we emit names but no declare intrinsics